docs/development/config_tests.rst

```diff
@@ -0,0 +1,103 @@
+Configuration Tests
+===================
+
+testinfra_ tests verify the end state of the vagrant machines. Any
+changes to the Ansible configuration should have a corresponding
+spectest.
+
+.. _testinfra: https://testinfra.readthedocs.io/en/latest/
+
+Installation
+------------
+
+.. code:: sh
+
+    pip install -r testinfra/requirements.txt
+
+Running the tests
+-----------------
+
+In order to run the tests, each VM will be created and provisioned, if
```

Comment:
Great docs. One note: the voice here makes it sound like the test suite will create the machine for you - it might be useful to make this more explicit, something like: "each VM will need to be created and provisioned - i.e. be sure you run `vagrant up <machine>` on the relevant machine before running the testinfra suite for that machine" (this exact language not necessary).